plugins/node/opentelemetry-instrumentation-pg/src/types.ts

```diff
@@ -27,12 +27,28 @@ export interface PgInstrumentationExecutionResponseHook {
   (span: api.Span, responseInfo: PgResponseHookInformation): void;
 }
 
+export interface QueryContext {
+  span: api.Span;
+  query?: string;
+  config?: NormalizedQueryConfig;
+  params?: unknown[];
+}
+
+export interface PgPostQueryHookFunction {
+  (ctx: QueryContext): void;
```

Comment:
Can you please change the hook signature to follow the other hook example in this file:
```js
export interface PgInstrumentationExecutionResponseHook {
  (span: api.Span, responseInfo: PgResponseHookInformation): void;
}
```

First parameter should be the span.
Second parameter is called `Info` on the other hook. Could be nice in my opinion to call it `queryInfo` (to stay consistent with the existing name. 
From what I remember, this is also the convention in other instrumentations.